PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs

```diff
@@ -279,11 +280,20 @@ public int GetPSBreakpointId(PowershellBreakpoint bp)
         /// <summary>
         /// Get runspace availability
         /// </summary>
+        /// <param name="executionPriority">Within execution priority, runspace will try to kill existing low priority task to free up runspace</param>
         /// <returns>runspace availability enum</returns>
-        public RunspaceAvailability GetRunspaceAvailability()
+        public RunspaceAvailability GetRunspaceAvailability(bool executionPriority)

```

Comment:
Not yet, I put a flag here considering later on we may have "(false) scenario" where intellisense itself tries to check runspace availability before invoking, coz we want to support intellisense during debugging.